setup.py

```diff
@@ -122,8 +122,8 @@
         'cachetools',
         'click>=5.0',
         'cloudpickle>=0.4',
-        'dask[array]<2024.11.0',  # Dask versions from 2024.11 cause problems with numpy2
-        'distributed<2024.11.0',
+        'dask[array]!=2024.11.*,!=2024.12.0',  # Dask versions >=2024.11 and < 2024.12.1 don't serialise Measurements
+        'distributed!=2024.11.*,!=2024.12.0',
```

Comment:
given the change to `Measurement` class is this constraint still needed?

Reply:
It might be, I'll test. 

Those versions had some additional breakage shown in https://github.com/dask/distributed/issues/8932 and https://github.com/opendatacube/datacube-core/issues/1656 . The example Emma included in the distributed issue was fixed in 2024.12.1, but ODC was still broken.

Reply:
Tested locally with 2024.11.1 and it works, so I've removed those version restrictions.